src/components/shared/BankData.vue

```diff
@@ -23,13 +27,20 @@ import { useI18n } from 'vue-i18n';
 
 interface Props {
 	bankTransferCode?: String;
+	bankTransferAmount?: Number
 }
 
-defineProps<Props>();
+const props = defineProps<Props>();
 
-const { t } = useI18n();
+const { t, n } = useI18n();
 const ibanParts = computed( () => t( 'bank_data_operator_account_iban' ).split( ' ' ) );
 
+const bankTransferFormattedAmount = computed<string>( ()=> t(
```

Comment:
We're already formatting the amount in `SuccessMessageBankTransfer` so you should pass into the `BankData` component that rather than formatting it twice. Something like this in `SuccessMessageBankTransfer`:

```js
const formattedAmount = n( props.donation.amount, { key: 'currency', currencyDisplay: 'name' } );

const donationSummaryMessage = computed<String>( () => {
	return t( 'donation_confirmation_payment_bank_transfer', {
		formattedAmount,
		interval: t( 'donation_confirmation_bank_transfer_interval_' + props.donation.interval ),
	} );
} );
```

Then in the template:

```xml
<BankData :bank-transfer-code="donation.bankTransferCode" :bank-transfer-amount="formattedAmount"/>
```